RELEASE_NOTES.md

```diff
@@ -1,8 +1,63 @@
 # Release Notes
 
+## stable-1.15-2.4.0, stable-1.16-2.4.0, stable-1.17-2.4.0
+
+* - Elasticsearch:
+    - Fixes regression from [helm/charts&#35;17643](https://github.com/helm/charts/pull/17643) where the explicit selectors do match the previously implicit selectors.
+    - Fix plugin install initcontainer which would fail if plugin already exists. This happens when Node reboots and keeps emptyDir, or if elasticsearch image already contains plugin
+    - In private environments where we replicate all the images, the test image cannot be pulled due it misses imagePullSecrets ([#497](https://github.com/mesosphere/kubernetes-base-addons/pull/497), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
```

Comment:
Didn't we revert this bc its broken? https://github.com/mesosphere/kubernetes-base-addons/pull/512

If the revert isn't included in the release, shouldn't it be? Its going to break upgrades.